assets/src/stories-editor/components/right-click-menu/index.js

```diff
@@ -0,0 +1,284 @@
+/* global navigator */
```

Comment:
Should not be needed anymore after #3150 I believe. Otherwise it sounds like a bug, and it should be added manually added to `.eslintrc`.